cornice_swagger/swagger.py

```diff
@@ -276,14 +283,18 @@ class DefinitionHandler(object):
 
     json_pointer = '#/definitions/'
 
-    def __init__(self, ref=0):
+    def __init__(self, ref=0, typ_dispatcher=TypeConversionDispatcher()):
```

Comment:
Hmm, I see. But I still think that passing the dispatcher is maybe not the best approach.

One solution that i think is a little more simple is to have a `type_converter_mapping={}` that update the default converters and `default_type_converter=None`. When the default converter is not None, we use it on missing cases instead of raising an exception. Thoughts?